src/__swaps__/utils/time.ts

```diff
@@ -0,0 +1,49 @@
+import parseMilliseconds from 'parse-ms';
+
+import * as i18n from '@/languages';
+
+const buildLocalizedTimeUnitString = ({ plural, short, unit }: { plural?: boolean; short: boolean; unit: string }) => {
+  const length = short ? 'short' : 'long';
+  const plurality = plural ? 'plural' : 'singular';
+
+  return i18n.t(`time.${unit}.${length}.${plurality}`);
```

Comment:
probably same here, should confirm we have the same values

Reply:
confirmed